pageserver/src/tenant_mgr.rs

```diff
@@ -74,8 +72,8 @@ pub enum TenantState {
     Broken,
 }
 
-impl fmt::Display for TenantState {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+impl std::fmt::Display for TenantState {
```

Comment:
Why this form is preferrable?

Reply:
No strong reason, can revert.
I usually keep "entities, used in one case" qualified and replace them to imports, when more usages come.

Reply:
Ok, I think thats just a matter of preference. I would rather keep it as it is to avoid larger diff, not insisting.

Maybe you know, is there a lint for something like that so we use similar style everywhere? Didnt find anything in clippy for that

Reply:
Alas, not sure there's one.